content/news/core-monthly/2023/core-monthly-2022-12-01-2022-12-31.md

```diff
@@ -0,0 +1,377 @@
+
+### PrestaShop 8.0.1
+
+The first patch version for PrestaShop 8 is on the horizon. With over [30 bug fixes](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Amerged+is%3Apr+milestone%3A8.0.1+label%3A%22Bug+fix%22) and [plenty of minor improvements](https://github.com/PrestaShop/PrestaShop/pulls?page=1&q=is%3Amerged+is%3Apr+milestone%3A8.0.1+label%3AImprovement) this version aims to increase the stability of the latest major release.
+
+I understand that some of you are waiting for the first patch to start thinking about the upgrade. The good news is that both the autoupgrade module and the patch versions are very very close to being fully available.
+
+### Autoupgrade module
+
+Speaking of upgrades, [autoupgrade v4.15 is now live](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.15.0). As I'm writing this article, PrestaShop 8 is not available in the module interface, but the module itself has everything needed for the upgrade to succeed. If you don’t see PrestaShop 8 as a new major release, you can upgrade to the latest version using the `Local archive` or `Local directory` option.
+
+The release, along with the .xml file required by the module, [is available on GitHub](https://github.com/PrestaShop/PrestaShop/releases/tag/8.0.0) and if you don’t know how to use these options, [you can watch a short demo](https://youtu.be/SBz7lApJ_pA?t=1063) of the process that I showed during the Public Demo.
+
+### Pull requests overview
+
+In December, there were over 200 merged pull requests. Let's dig into some of the most interesting contributions that target PrestaShop 8.1 and module updates.
+
+**Enabling profiler in back office**
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), in PrestaShop 8.1, it will be possible to enable performance metrics in the front office directly from the back office. There's still some work to do regarding this feature, as the community feedback was to add an extra security layer to let only shop maintainers see the results of profiling. You can see the [details in the pull request](https://github.com/PrestaShop/PrestaShop/pull/30603).
+
+**Update from Vue 2 to Vue 3**
+
+Thanks to [@NeOMakinG](https://github.com/NeOMakinG), core components written in Vue 2 are now fully migrated to Vue 3. You can see [this massive pull request](https://github.com/PrestaShop/PrestaShop/pull/28463) to see the migration details. Kudos to [@NeOMakinG](https://github.com/NeOMakinG) for such amazing work!
+
+**Making PrestaShop core better**
+
+In every release, project members try to improve the software's core. This time I present you a pull request where [@FabienPapet](https://github.com/FabienPapet) migrated `Tools::str2url` so it uses the `StringModifier` utility class. Thanks to the migration, the function is now fully unit-tested. With such modifications, the project becomes more stable and reliable. You can see all the [implementation details in the pull request](https://github.com/PrestaShop/PrestaShop/pull/30357).
```

Comment:
```suggestion
Every release includes minor changes that improve the software's core in some way or another. This time I present you a pull request where [@FabienPapet](https://github.com/FabienPapet) refactored `Tools::str2url` so that it uses the `StringModifier` utility class. Thanks to his work, the function is now fully unit-tested, and the project has become more stable and reliable. You can see all the [implementation details in the pull request](https://github.com/PrestaShop/PrestaShop/pull/30357).
```